task/09-functions-n-closures-tasks.js

```diff
@@ -132,10 +144,20 @@ function retry(func, attempts) {
  *
  */
 function logger(func, logFunc) {
-    throw new Error('Not implemented');
+  return function () {
+    let funcName = `${func.name}(${JSON.stringify(Array.from(arguments)).slice(
+      1,
```

Comment:
In this case, the "slice" method parameters will be more readable on the same line.